tools/build_config/clone_herbert_branch.ps1

```diff
@@ -0,0 +1,26 @@
+# Clone and build the given Herbert branch. If no branch is given, default to
+# master
+# Pass a specific branch using the -branch parameter, any other arguments are
+# passed verbatim to the Herbert build script
+
+param(
+  [string][Alias("B")]$branch = "master",
```

Comment:
your API is inconsistent internally and between the bash and powershell scripts;
- the bash script doesn't accept short-args 
- you don't have a short form of the second argument.

Reply:
I've just removed the short argument. It's not used and this is not a public facing script